packages/relay-service/lib/index.js

```diff
@@ -0,0 +1,60 @@
+import assert from 'assert';
+import axios from 'axios';
+import {generatePayload, toBase64} from './payloads';
+
+let serviceURL = 'https://relay.dock.io';
```

Comment:
can we read this from a config? Thinking at some point we may want to be able to change it (test env, white label clients running their own, etc.)